src/Exception/NotFoundException.php

```diff
@@ -5,8 +5,10 @@
 /**
  * Thrown when a discovery does not find any matches.
  *
+ * @final
+ *
  * @author Márk Sági-Kazár <mark.sagikazar@gmail.com>
  */
-final class NotFoundException extends \RuntimeException
+/*final */class NotFoundException extends \RuntimeException
```

Comment:
ah, makes sense. maybe add that comment after the `@final` then?i think explanations always are a good thing.